### PR TITLE
Update local-ai readme.md

### LINKED ISSUE #5602: local-ai readme update, depericated URL listed
### Is your feature request related to a problem? Please describe. 

No issue, just noticed that my setup of local-ai was no longer fetching a model for 'gpt4all-j'

### Describe the solution you'd like

Fix documentation

### Additional context

Per the suggested github repository in the readme for the local-ai container: https://github.com/nextcloud/all-in-one/blob/main/community-containers/local-ai/readme.md

```Note: this Repository is deprecated. The model gallery has now moved to https://github.com/mudler/LocalAI/tree/master/gallery and is configured by default in LocalAI.``` https://github.com/go-skynet/model-gallery/tree/main?tab=readme-ov-file#note-this-repository-is-deprecated-the-model-gallery-has-now-moved-to-httpsgithubcommudlerlocalaitreemastergallery-and-is-configured-by-default-in-localai

The URL suggested should be updated to the new location.

### PR DESCRIPTION
**Pull request by ShapeShifter499**
Close #5602

### CODE REVIEW

**Comment by ShapeShifter499** [comment]
gpt4all-j and whisper seems to have been added as default modules for LocalAI, so I have removed those lines. Please correct me if I'm wrong.

**Comment by ShapeShifter499** [comment]
This pull request might not be required, depending on the discussion of https://github.com/szaimen/aio-local-ai/issues/48

File: community-containers/local-ai/readme.md [hunk 1]
```diff
@@ -3,20 +3,12 @@ This container bundles Local AI and auto-configures it for you.
...
+- After the container was started the first time, you should see a new `nextcloud-aio-local-ai` folder when you open the files app with the default `admin` user. In there you should see a `models.yaml` config file. You can now add models in there. Please refer [here](https://github.com/mudler/LocalAI/blob/master/gallery/index.yaml) where you can get further urls that you can put in there. Afterwards restart all containers from the AIO interface and the models should automatically get downloaded by the local-ai container and activated.
 - Example for content of `models.yaml` (if you add all of them, it takes around 10GB additional space):
 ```yaml
 # Stable Diffusion in NCNN with c++, supported txt2img and img2img 
-- url: github:go-skynet/model-gallery/stablediffusion.yaml
+- url: github:mudler/LocalAI/blob/master/gallery/stablediffusion.yaml
```
Comment (szaimen) [inline community-containers/local-ai/readme.md:10]: Is blob/master actually needed?
```suggestion
- url: github:mudler/LocalAI/gallery/stablediffusion.yaml
```

**Review by szaimen — COMMENTED** [review]
Hi, thanks for your PR! 

Just one comment below

**Comment by ShapeShifter499** [comment]
> Hi, thanks for your PR!
> 
> Just one comment below

Updated!

**Review by szaimen — APPROVED** [review]
LGTM, thanks! 😊

**Pull request merged**